duckduckgo/iOS
# Reload page when content blocker settings change

Reviewer: Chris
Asana: https://app.asana.com/0/361428290920652/375790519910150

**Description**:
Reloads the page when the user changes their content blocker settings.

**Steps to test this PR**:
1. Navigate to a page with trackers
2. Switch content blocking off/on globally and off/on at the domain level
3. The page will reload each time. Watch the logs and ensure they detect trackers but only block when setting is on

###### Reviewer Checklist:


###### PR DRI Checklist:
